spec/spec_helper.rb

```diff
@@ -2,14 +2,16 @@
 
 $LOAD_PATH.unshift File.expand_path('../lib', __dir__)
 require 'cronitor'
-require 'webmock/rspec'
-require 'support/fake_cronitor'
-WebMock.disable_net_connect! allow_localhost: true
+# require 'webmock/rspec'
+# require 'support/fake_cronitor'
+# WebMock.disable_net_connect! allow_localhost: true
```

Comment:
If we're not using this anymore, may as well simply delete these lines. We've got versions, after all!